packages/synchro-charts/src/components/charts/sc-webgl-base-chart/renderAxis.ts

```diff
@@ -68,8 +68,9 @@ const xAxisConstructor = (size: SizeConfig, viewport: ViewPort) => {
 const yAxisConstructor = (size: SizeConfig, viewport: ViewPort) => {
   const { yTickCount } = tickCount(size);
   const { yScale } = scales(size, viewport);
+
   return axisLeft(yScale)
-    .ticks(yTickCount)
+    .ticks(yTickCount, 's')
```

Comment:
Cool :). Looks like d3-format has a whole bunch of options - https://github.com/d3/d3-format#locale_formatPrefix. 

Is there a reason you chose this particular format vs. exponential which is used on chart tooltips? (https://github.com/awslabs/synchro-charts/issues/66)

Reply:
exponential feels significantly harder to read, while decimal notation with SI prefix is something most of us are used to seeing 
ie: `50k` vs `5.0e+4`